src/coverlet.core/Helpers/InstrumentationHelper.cs

```diff
@@ -14,6 +14,13 @@ namespace Coverlet.Core.Helpers
 {
     internal static class InstrumentationHelper
     {
+        private static readonly Dictionary<string, string> _backupList = new Dictionary<string, string>();
```

Comment:
sooner or later we should remove al these statics and use msbuild input/output https://github.com/tonerdo/coverlet/issues/364